twirssi.pl

```diff
@@ -322,10 +322,10 @@ sub format_expand {
     my %args = @_;
     $args{fmt} =~ s/\$n/\@$args{nick}/g;
     if (defined $args{data} and $args{data} ne '') {
-        $args{fmt} =~ s/\${|\$}//g;
+        $args{fmt} =~ s/\$\{|\$\}//g;
```

Comment:
I don't mind escaping the `{`, but I don't think they have any special meaning except when they contain digits?